bears/vcs/VCSCommitBear.py

```diff
@@ -0,0 +1,107 @@
+import abc
+import re
+
+from coalib.bears.GlobalBear import GlobalBear
+from coalib.results.HiddenResult import HiddenResult
+from coalib.results.Result import Result
+from coalib.misc.Shell import run_shell_command
+from collections import namedtuple
+from coala_utils.decorators import (enforce_signature, generate_ordering,
+                                    generate_repr)
+
+
+@generate_repr(('id', hex),
+               'origin',
+               'commit_info',
+               'message')
+@generate_ordering('commit_info',
+                   'contents',
+                   'severity',
+                   'confidence',
+                   'origin',
+                   'message_base',
+                   'message_arguments',
+                   'aspect',
+                   'additional_info',
+                   'debug_msg')
+class CommitResult(HiddenResult):
+
+    @enforce_signature
+    def __init__(self, origin, commit_info):
+        Result.__init__(self, origin, 'HEAD commit information', commit_info)
```

Comment:
Delete `commit_info` argument from this

Reply:
Delete the whole `Result` line inside `_init_`. I dont understand its existence. you are already passing `commit_info` as contents right :)

Reply:
>Well, the reason of why Result.__init__ is there is because HiddenResult is a subclass of Result.
So we need to run the Result.__init__

This makes sense. Well keep it then.